src/main/java/com/lpvs/service/LPVSQueueService.java

```diff
@@ -147,6 +147,7 @@ public BlockingDeque<LPVSQueue> getQueue() {
      * @throws InterruptedException If interrupted while processing the queue.
      */
     public void checkForQueue() throws InterruptedException {
+        QUEUE.clear();
```

Comment:
As an option, I'd pur warning here if queue is not clear. Cause we should pay attention, some webhooks are not processed.

Reply:
I don't think that warning is needed here. This is the normal operation of the LPVS queue.